css/dashboard.css

```diff
@@ -315,25 +385,35 @@ li {
 
 
 /* Tables
-�������������������������������������������������� */
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
 table {
   border-collapse: collapse;
 }
 th,
-td {
-  padding: 12px 15px;
-  text-align: left;
-  border-bottom: 1px solid #E1E1E1; }
-th:first-child,
-td:first-child {
-  padding-left: 0; }
-th:last-child,
-td:last-child {
-  padding-right: 0; }
+tr {
```

Comment:
This seems to be missing the fix from this solution
https://stackoverflow.com/questions/47575949/css-hover-formats-table-rows-unpredictably-in-firefox
Can't see a row with 
```css
tr.notselected {
      border: 0px solid black; 
   }
```